python/unityagents/brain.py

```diff
@@ -1,5 +1,5 @@
 class BrainInfo:
-    def __init__(self, observation, state, memory=None, reward=None, agents=None, local_done=None):
+    def __init__(self, observation, state, memory=None, reward=None, agents=None, local_done=None, action =None):
```

Comment:
Should this be something like `previous_action` so that it is clear that it doesn't correspond to the current state?